cmd/minishift/cmd/start.go

```diff
@@ -238,6 +238,10 @@ func handleProxies() *util.ProxyConfig {
 
 func getSlice(key string) []string {
 	if viper.IsSet(key) {
+		value := viper.Get(key)
+		if _, ok := value.(string); ok {
+			return nil
+		}
```

Comment:
@praveenkumar I did some testing around this and it seems, when we pass --docker-env to `minishift start` getSlice is returning nil i.e. viper is not considering it as a string slice. So this method is useful when we read from config.json. Can you please add comments communicating the same for the function?